python/doc/examples/probabilistic_modeling/overview_univariate_distributions.ipynb

```diff
@@ -0,0 +1,602 @@
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "distribution = ot.Beta(2.5, 5, -1, 2)\n",
+    "distribution.drawPDF()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `BetaMuSigma` class provides another parametrization, based on the expectation $\\mu$ and the standard deviation  $\\sigma$ of the random variable:\n",
+    "\n",
+    "$$\n",
+    "\\mu = a + \\frac{(b-a)r}{t}\n",
+    "$$\n",
+    "\n",
+    "and\n",
+    "\n",
+    "$$\n",
+    "\\sigma^2 = \\left(\\frac{b-a}{t}\\right)^2 \\frac{r(t-r)}{t+1}.\n",
+    "$$\n",
+    "\n",
+    "Inverting the equations, we get:\n",
+    "\n",
+    "$$\n",
+    "t = \\frac{(b-\\mu)(\\mu-a)}{\\sigma^2} - 1\n",
+    "$$\n",
+    "\n",
+    "and\n",
+    "\n",
+    "$$\n",
+    "r = \\frac{\\mu-a}{b-a} t.\n",
+    "$$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following session creates a beta random variable with parameters $\\mu=0.2$, $\\sigma=0.6$, $a=-1$ et $b=2$."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<p>[2,5,-1,2]</p>"
+      ],
+      "text/plain": [
+       "class=Point name=Unnamed dimension=4 values=[2,5,-1,2]"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "parameters = ot.BetaMuSigma(0.2, 0.6, -1, 2)\n",
+    "parameters.evaluate()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `ParametrizedDistribution` creates a distribution from a parametrization."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<p>class=ParametrizedDistribution parameters=class=BetaMuSigma name=Unnamed mu=0.2 sigma=0.6 a=-1 b=2 distribution=class=Beta name=Beta dimension=1 r=2 t=5 a=-1 b=2</p>"
+      ],
+      "text/plain": [
+       "class=ParametrizedDistribution parameters=class=BetaMuSigma name=Unnamed mu=0.2 sigma=0.6 a=-1 b=2 distribution=class=Beta name=Beta dimension=1 r=2 t=5 a=-1 b=2"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "param_dist = ot.ParametrizedDistribution(parameters)\n",
+    "param_dist"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Functions of distributions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "subslide"
+    }
+   },
+   "source": [
+    "The library provides also algebra of univariate distributions:\n",
+    "\n",
+    " - +, -\n",
+    " - *, /\n",
+    "\n",
+    "It provides also some functions to get the full distributions of `f(x)` where `f` can be equal to :\n",
```

Comment:
"Also" should be before "provides". Please have a look at the alternative suggestion below before accepting this one.
```suggestion
    "It also provides some functions to get the full distributions of `f(x)` where `f` can be equal to :\n",
```

Reply:
Ok. Fixed.